### DISCUSSION TITLE
Spelling error in "Update indexer metadata" UI block

### DISCUSSION THREAD

**Discussion by rost1989** [subquery/subql#1008] [Frontier TestNet - Bug Reporting]
There is a spelling error in "Update indexer metadata" in the word "valid".
![image](https://user-images.githubusercontent.com/32965628/167723736-b5eca468-cbd2-47ef-b8f8-e0edfc83371e.png)

**Comment by mzxyz** [comment]
Thanks for point out this

**Comment by mzxyz** [comment]
Resolved in `coordinator:v0.1.10`
